app/views/kaminari/_gap.html.erb

```diff
@@ -0,0 +1,8 @@
+<%# Non-link tag that stands for skipped pages...
+  - available local variables
+    current_page:  a page object for the currently displayed page
+    total_pages:   total number of pages
+    per_page:      number of items to fetch per page
+    remote:        data-remote
+-%>
+<%# //NOSONAR -%><li class="moj-pagination__item">&hellip;</li>
```

Comment:
I've had to turn off sonar for these custom kaminari view files, since it flags them with the following bug:

`"<li>" and "<dt>" item tags should be in "<ul>", "<ol>" or "<dl>" container tags`.

In fact, these are! Sonar just can't tell since they're not rendered directly (kaminari does that, inside the `<ul>` in the `/paginator.html.erb` partial.

<img width="528" alt="Screenshot 2021-02-11 at 16 03 01" src="https://user-images.githubusercontent.com/18436946/107664867-c1552e00-6c84-11eb-8e8e-8b67428b38ee.png">